oxide-auth/src/primitives/registrar.rs

```diff
@@ -137,7 +137,7 @@ pub struct EncodedClient {
     pub client_id: String,
 
     /// The registered redirect uri.
-    pub redirect_uri: Url,
```

Comment:
I'll have to think if this is the best way to approach this, possibly better to add a new attribute.